src/components/skeletons/ElementSkeleton/ElementSkeleton.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+import Shimmer from '../Shimmer'
+
+import './element-skeleton.scss'
+
+type ElementVariant = 'text' | 'button' | 'image' | 'badge'
+
+interface Props {
+  shimmer?: boolean
+  type: ElementVariant
+}
+
+function ElementSkeleton({ type, shimmer = false }: Props) {
```

Comment:
How about calling it `SkeletonElement`? It looks more like the pattern we have been following 🤔

Reply:
I agree with your suggestion, I'll rename all of them